packages/client-sdk-web/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "prebuild": "eslint . --ext .ts",
     "test": "jest",
-    "unit-test": "jest unit",
+    "unit-test": "jest unit --passWithNoTests",
```

Comment:
why is this necessary?  If there is a file that doesn't have a test in it can we just delete that file?

Reply:
Hm, actually, I think I need to put it back because the `build-and-test-package` script is explicitly calling it. This allows that script to call `unit-test` and allows `unit-test` to no-op.

Reply:
Added back in 63af729 to support `build-and-test-package` script.

Reply:
I don't like the passWithNoTests flag.  Do we not have any unit tests?  Could we just add a "hello world" one that asserts true == true?

I have seen issues in the past where someone misconfigured a test source dir and then a flag like this would cause the build to pass even if there were tons of failing unit tests because they didn't actually end up getting run.

Reply:
to be clear - i want to keep the unit tests directive but i want to get rid of that flag :)

Reply:
Yeah, that's fair. I added a no-op unit test in b579c3d.